### PR TITLE
Update platforms.md

### PR DESCRIPTION
**Pull request by Andreyogld3d**
QNX, BSD Unix, Fuchsia

### CODE REVIEW

File: chapters/platforms.md [hunk 1]
```diff
@@ -20,6 +20,18 @@ Vulkan is not natively supported on iOS, but can still be targeted with [Vulkan
 
 Vulkan is supported on many Linux distributions.
 
+## BSD Unix
+
+Vulkan is supported on many BSD Unix distributions.
+
+## QNX
+
+Vulkan is supported on QNX operation system, using [VK_QNX_screen_surface](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/vkCreateScreenSurfaceQNX.html)
```
Comment (sfricke-samsung) [inline chapters/platforms.md:29]: all looks good, but should just move the second part to
https://github.com/KhronosGroup/Vulkan-Guide/blob/master/chapters/wsi.md#surface

File: chapters/platforms.md [hunk 2]
```diff
@@ -20,6 +20,18 @@ Vulkan is not natively supported on iOS, but can still be targeted with [Vulkan
...
+
+## Fuchsia
+
+Vulkan is supported on Fuchsia operation system.
```
Comment (sfricke-samsung) [inline chapters/platforms.md:33]:
```suggestion
Vulkan is supported on the [Fuchsia operation system](https://fuchsia.dev/fuchsia-src/concepts/graphics/magma/vulkan).
```

File: chapters/platforms.md [hunk 3]
```diff
@@ -20,6 +20,18 @@ Vulkan is not natively supported on iOS, but can still be targeted with [Vulkan
 
 Vulkan is supported on many Linux distributions.
 
+## BSD Unix
```
Comment (sfricke-samsung) [inline chapters/platforms.md:23]: So small nit to add these alphabetically with how the rest are

**Review by Andreyogld3d — COMMENTED** [review]
Ok

**Review by Andreyogld3d — COMMENTED** [review]
OK

**Comment by sfricke-samsung** [comment]
Thanks again for helping keep the page up-to-date!
looks all good to me,

**Pull request merged**